data/software_stack/eessi.yaml

```diff
@@ -0,0 +1,47 @@
+jupyterhub::kernel::setup: venv
+jupyterhub::kernel::venv::python: /cvmfs/pilot.eessi-hpc.org/2020.10/init/eessi_pilot_python3
```

Comment:
This now requires https://github.com/EESSI/software-layer/pull/43 (which is a wrapper to ensure we land on the right Python 3 for the right architecture)